docs/about/compatibility/openmp.md

```diff
@@ -77,8 +77,7 @@ Obtain the value of `gpu-arch` by running the following command:
 
 [//]: # (dated link below, needs updating)
 
-See the complete list of compiler command-line references
-[here](https://github.com/ROCm/llvm-project/blob/amd-stg-open/clang/docs/CommandGuide/clang.rst).
+See the complete list of [compiler command-line references](https://github.com/ROCm/llvm-project/blob/amd-staging/openmp/docs/CommandLineArgumentReference.rst).
```

Comment:
Should this be a reference to the published docs instead of the GitHub repo?

Reply:
Unfortunately, we don't have a published page for openmp to link to yet.  So the stop-gap right now, is to point to the GH file.